test/ExecCommand.js

```diff
@@ -125,6 +125,7 @@ describe("ExecCommand", () => {
             env: expect.objectContaining({
               LERNA_PACKAGE_NAME: "package-2",
             }),
+            reject: true,
```

Comment:
We test yargs usage in the integration tests, not unit tests. I've often thought of writing an arg bootstrapper to streamline this.